1d-burgers/inf_cont_burgers.py

```diff
@@ -28,16 +28,20 @@
     # Data size on the solution u
     hp["N_u"] = 100
     # Collocation points size, where we’ll check for f = 0
-    hp["N_f"] = 10000
+    # [mtsokol] for bigger values slows down drastically
+    hp["N_f"] = 100
     # DeepNN topology (2-sized input [x t], 8 hidden layer of 20-width, 1-sized output [u]
-    hp["layers"] = [2, 20, 20, 20, 20, 20, 20, 20, 20, 1]
+    # [mtsokol] 7x7=49 for one_hot_encoding, more about it in neuralnetwork.py
+    # [mtsokol] IMPORTANT! when modifying input size remember to modify BSplineLayer(num_elem=) param. (Should be done in one place only. Will fix it)
+    # [mtsokol] num_elem = sqrt(layers[0]) - 2 (only for one_hot_encoding)
+    hp["layers"] = [49, 20, 20, 20, 20, 20, 20, 20, 20, 1]
```

Comment:
Have you tried to lower the number of layers? It has a big impact on computational cost. Maybe you could try say 2 layers of 50 and a larger N_f

Reply:
Thank you for your comments!
So I tried with only two layers, different input magnitude but it still just pull coefficients close to zero. 
I've checked bspline logic once again for errors (same as in notebook which I linked) and it seems fine so I'm not sure if such procedure can converge to desired solution. 

Should I commit those layer changes?

Reply:
No don't commit. But was the performance better?

Reply:
Slightly better but without significant improvement.